src/bpftrace.cpp

```diff
@@ -38,6 +38,7 @@ namespace bpftrace {
 
 DebugLevel bt_debug = DebugLevel::kNone;
 bool bt_verbose = false;
+volatile sig_atomic_t bt_sigint_called = false;
```

Comment:
As stated by @ajor, I think would be preferable to have this variable as a static member of BPFtrace class...